Makefile

```diff
@@ -20,15 +20,10 @@ $(call help,make dev,run the whole app \(all workers\))
 dev: python
 	@pyenv exec tox -qe dev
 
-.PHONY: web
-$(call help,make web,run just a web worker)
-web: python
-	@pyenv exec tox -qe dev --run-command 'gunicorn --bind :4000 --workers 1 --reload --timeout 0 --paste conf/development.ini'
-
 .PHONY: shell
 $(call help,make shell,"launch a Python shell in this project's virtualenv")
 shell: python
-	@pyenv exec tox -qe dev --run-command 'pshell conf/development.ini'
```

Comment:
`make shell` is broken on `main` currently: `pshell` is a Pyramid thing. The `pyapp` cookiecutter's `make shell` just runs `ipython` directly